<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KCreditCardSchedulePage</class>
  <widget class="QWidget" name="KCreditCardSchedulePage">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>544</width>
        <height>391</height>
      </rect>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <widget class="QLabel" name="qLabel16">
          <property name="text">
            <string>For credit card accounts you can create a monthly schedule which will automatically remind you about the payment. Please fill in the details about this payment here.</string>
          </property>
          <property name="wordWrap">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QCheckBox" name="m_reminderCheckBox">
          <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
          </property>
          <property name="text">
            <string>Yes, remind me when the payment is due</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QFrame" name="m_paymentFrame">
          <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Plain</enum>
          </property>
          <layout class="QGridLayout">
            <property name="margin">
              <number>0</number>
            </property>
            <item row="0" column="0">
              <widget class="QLabel" name="textLabel2">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>180</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="text">
                  <string>Name of schedule</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="KLineEdit" name="m_name"/>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="textLabel3">
                <property name="text">
                  <string>Payee</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="1">
              <widget class="KMyMoneyPayeeCombo" name="m_payee"/>
            </item>
            <item row="2" column="0">
              <widget class="QLabel" name="TextLabel2_2">
                <property name="text">
                  <string>Estimated monthly payment</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="1">
              <widget class="AmountEdit" name="m_amount">
                <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                </property>
              </widget>
            </item>
            <item row="3" column="0">
              <widget class="QLabel" name="TextLabel4_3">
                <property name="text">
                  <string>Due date of next payment</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="1">
              <widget class="KMyMoneyDateInput" name="m_date">
                <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                </property>
              </widget>
            </item>
            <item row="4" column="0">
              <widget class="QLabel" name="TextLabel3_3">
                <property name="text">
                  <string>Payment should be made
from account</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="1">
              <widget class="KMyMoneyCategory" name="m_paymentAccount"/>
            </item>
            <item row="5" column="0">
              <widget class="QLabel" name="textLabel4">
                <property name="text">
                  <string>Payment method</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="5" column="1">
              <widget class="KMyMoneyGeneralCombo" name="m_method"/>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="spacer23">
          <property name="sizeHint">
            <size>
              <width>20</width>
              <height>31</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
    <connections>
    <connection>
      <sender>m_reminderCheckBox</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_paymentFrame</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
  </connections>
  <customwidgets>
  <customwidget>
   <class>KMyMoneyGeneralCombo</class>
   <extends>QWidget</extends>
   <header>kmymoneygeneralcombo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KMyMoneyPayeeCombo</class>
   <extends>QWidget</extends>
   <header>kmymoneypayeecombo.h</header>
   <container>1</container>
  </customwidget>
    <customwidget>
      <class>KLineEdit</class>
      <extends>QLineEdit</extends>
      <header location="global">klineedit.h</header>
    </customwidget>
    <customwidget>
      <class>AmountEdit</class>
      <extends>QLineEdit</extends>
      <header location="global">amountedit.h</header>
    </customwidget>
  </customwidgets>
</ui>
